prompting/validators/reward/diversity.py

```diff
@@ -173,9 +174,9 @@ def get_rewards(
 
         # Return all
         if historic_rewards != None:
-            return batch_rewards * historic_rewards
+            return {'reward': batch_rewards * historic_rewards}
```

Comment:
Are we returning a raw dict here? What if we use the `BaseRewardModel` to streamline the behavior of get_rewards ->  `BaseRewardEvent` or any derivative class

Reply:
just fixed! thank you:)